src/llmcompressor/transformers/finetune/text_generation.py

```diff
@@ -83,21 +83,6 @@ def eval(**kwargs):
     main(model_args, data_args, recipe_args, training_args)
 
 
-def oneshot(**kwargs):
```

Comment:
Should we retain this with a deprecated warning? This is a pretty commonly used function

Reply:
> +1